lib/keyValueStorageClient.js

```diff
@@ -2,18 +2,15 @@ import Keyv from 'keyv';
 import { config } from '../config.js';
 
 const storageConnectionString = config.get('storageConnectionString');
-
-if (!config.get('storageConnectionString')) {
-    console.log('No storageConnectionString specified. Please set the storageConnectionString or STORAGE_CONNECTION_STRING environment variable if you need caching or stored context.')
-}
+const cortexId = config.get('cortexId');
 
 // Create a keyv client to store data
 const keyValueStorageClient = new Keyv(storageConnectionString, {
     ssl: true,
     abortConnect: false,
     serialize: JSON.stringify,
     deserialize: JSON.parse,
-    namespace: 'cortex-context'
+    namespace: `${cortexId}-cortex-context`
```

Comment:
would not this cause context based info get missed

Reply:
I just thought it would be easier to manage multiple deployments sharing one redis by giving them a unique namespace.  It will ignore stuff that’s already in Redis but I don’t think we’re currently storing anything in there?